WrfTrigger.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python3
+
+# Download Rain cell files, download Mean-ref files.
+# Check Rain fall files are available in the bucket.
+
+import fnmatch
+import sys
+import os
+import json
+import getopt
+import datetime
+from google.cloud import storage
+from curw.rainfall.wrf.extraction import observation_utils as utils
+from curwmysqladapter import MySQLAdapter, Data
+
+
+def usage():
+    usage_text = """
+Usage: ./CSVTODAT.py [-d YYYY-MM-DD] [-t HH:MM:SS] [-h]
+
+-h  --help          Show usage
+-d  --date          Date in YYYY-MM-DD. 
+-t  --time          Time in HH:00:00.
+-f  --forward       Future day count
+-b  --backward      Past day count
+-T  --tag           Tag to differential simultaneous Forecast Runs E.g. wrf1, wrf2 ...
+    --wrf-rf        Path of WRF Rf(Rainfall) Directory. Otherwise using the `RF_DIR_PATH` from CONFIG.json
+    --wrf-kub       Path of WRF kelani-upper-basin(KUB) Directory. Otherwise using the `KUB_DIR_PATH` from CONFIG.json
+"""
+    print(usage_text)
+
+def download_wrf_files():
```

Comment:
@hasithadkr7 it's better to have pure functions. Rather using global variables s.t. key_file, bucket_name etc. Otherwise specifically pass them as parameters. Please refer to https://stackoverflow.com/questions/37852630/best-practice-for-returning-a-variable-that-exists-in-global-scope